common/test/views/support/cleaner/AmpEmbedCleanerTest.scala

```diff
@@ -139,77 +182,177 @@ class AmpEmbedCleanerTest extends FlatSpec with Matchers {
 
 
   /////////////////////////////
```

Comment:
If you are doing block comments like this it's be better to use `/* */` commenting syntax https://wiki.scala-lang.org/display/SW/Syntax